genai-perf/README.md

```diff
@@ -377,6 +375,13 @@ the inference server.
 
 </br>
 
+### Telemetry Metrics
```

Comment:
Note that I did not provide example metrics by design. That would make the README much longer for this one section. 

If we want to provide example output, we should do it in its own tutorial that goes more in depth. We should also weigh doing so versus any extra maintenance required.

Reply:
I would say that we need that doc.
Not gating for this PR but we should make that ticket to make sure we follow up.

Reply:
@lkomali Would you be able to make that Jira ticket? If you could include the details about what a tutorial for Telemetry should include, that'd be helpful (e.g. if we're expanding it past Triton, is it different for Triton vs vLLM).

Reply:
Created TPA-895 to track this work.

Reply:
Sorry, I didn't check your comment yesterday. Thanks for creating the ticket @dyastremsky.